src/main/java/com/microsoft/sqlserver/jdbc/SQLServerADAL4JUtils.java

```diff
@@ -0,0 +1,55 @@
+package com.microsoft.sqlserver.jdbc;
+
+import java.net.MalformedURLException;
+import java.text.MessageFormat;
+import java.util.concurrent.ExecutionException;
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
+import java.util.concurrent.Future;
+
+import com.microsoft.aad.adal4j.AuthenticationContext;
+import com.microsoft.aad.adal4j.AuthenticationException;
+import com.microsoft.aad.adal4j.AuthenticationResult;
+import com.microsoft.sqlserver.jdbc.SQLServerConnection.ActiveDirectoryAuthentication;
+import com.microsoft.sqlserver.jdbc.SQLServerConnection.SqlFedAuthInfo;
+
+class SQLServerADAL4JUtils {
```

Comment:
Are you sure you do not want public class? By default this will use package level visibility.